script.js

```diff
@@ -0,0 +1,746 @@
+                        app.display.regionsArray.shift();
+                        if (laneData) {
+                            url = "bikelanes/" + laneData;
+                            app.laneAjaxQueue(url, region);
+                        } else {
+                            console.log("=== ALERT: Missing data for", region);
+                        }
+                    } else {
+                        app.display.regionsArray = ["AL", "AR", "DC", "MO", "PG"];
+                    }
+                }
+            }
+        }).fail(function(){
+            console.log("*** ajax fail T ***");
+        });
+    },
+
+    // ======= bufferAjaxQueue =======
+    bufferAjaxQueue: function(url, region, buffer) {
+        console.log("bufferAjaxQueue");
+        $.ajax({
+            url: url,
+            method: "GET",
+            dataType: "text"
+        }).done(function(jsonData){
+            console.log("*** ajax success ***");
+            var jsonData2 = $.parseJSON(jsonData);
+            var bufferFeatures = L.mapbox.featureLayer(jsonData2).addTo(app.activeMap);
+            app.state[region].bufferLayers[buffer] = bufferFeatures;
+            bufferFeatures.setStyle(app.display.bufferStyle);
+        }).fail(function(){
+            console.log("*** ajax fail T ***");
+        });
+    },
+
+    // ======= allLanesQueue =======
+    allLanesQueue: function(url, regionsArray) {
+        console.log("allLanesQueue");
+        if ((Array.isArray(regionsArray)) && (regionsArray.length > 0)) {
+            var region = regionsArray[0];
+        } else {
+            var region = regionsArray;
+        }
+
+        $.ajax({
+            url: url,
+            method: "GET",
+            dataType: "text"
+        }).done(function(jsonData){
+            console.log("*** ajax success ***");
+            var jsonData2 = $.parseJSON(jsonData);
+            var laneFeatures = L.mapbox.featureLayer(jsonData2).addTo(app.activeMap);
+            app.state[region].laneLayers.push(laneFeatures);
+            laneFeatures.setStyle(app.state[region].bikeLaneStyle);
+            if (loopCount < app.state[region].laneData.length - 1) {
+                loopCount++;
+                var laneData = app.state[region].laneData[loopCount];
+                app.allLanesQueue(url, region);
+            } else {
+                var arrayR = app.display.regionsArray;
+                arrayR.shift();
+                app.display.regionsArray = arrayR;
+                app.allLanesQueue(url, arrayR);
+            }
+        }).fail(function(){
+            console.log("*** ajax fail T ***");
+        });
+    },
+
+    // ======= initMap =======
+    initMap: function () {
+        console.log("app.initMap");
+        L.mapbox.accessToken = "pk.eyJ1IjoiYWx1bHNoIiwiYSI6ImY0NDBjYTQ1NjU4OGJmMDFiMWQ1Y2RmYjRlMGI1ZjIzIn0.pngboKEPsfuC4j54XDT3VA";
+        var map = L.mapbox.map(
+            app.map.mapEl,
```

Comment:
It's considerably more confusing and adds extra complexity to declare the settings for this map at the top in lines 26-36 rather than inline here when first creating the map. This is ok for this current PR but we should refactor to be less confusing and streamlined.